cs-config/cs_config/functions.py

```diff
@@ -125,9 +125,25 @@ def run_model(meta_param_dict, adjustment):
     iit_mods = convert_adj(adjustment[
         "Individual and Payroll Tax Parameters"],
                            meta_params.year.tolist())
+    filtered_ccc_params = {}
```

Comment:
@hdoupe Let me know if you think that this is not a very efficient way to handle the case where some parameters input on CS should be the same between the baseline and reform runs.

Reply:
I think this is the best you can do without creating a separate assumptions section, kind of like the Behavior section in Tax-Brain. You should get better lookup performance if you make `constant_param_list` a set. Other than that, this looks good to me.

Reply:
Ok - thanks @hdoupe.